ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -60,14 +71,27 @@
 
   private final File tempDir;
   private final StoreMetrics metrics;
+  private Pair<StoreKey, PersistentIndex.IndexEntryType> resetKey;
+  private Long lastModifiedTime;
```

Comment:
Why is this a class variable? Might be missing it but can you point towards its usage as a class variable?